libbeat/docs/processors-using.asciidoc

```diff
@@ -761,3 +762,33 @@ The fields added to the event are looking as following:
 -------------------------------------------------------------------------------
 
 NOTE: The host information is refreshed every 5 minutes.
+
+[[dissect]]
+=== Dissect
+
+The `dissect` processor allows you to use the `dissect` syntax to tokenizer incoming string into
```

Comment:
It sounds like you're planning more work for the docs, so I won't make too many comments. But I would shorten this to say:

The `dissect` processor tokenizes incoming strings....